chord_metadata_service/phenopackets/serializers.py

```diff
@@ -167,31 +120,108 @@ class Meta:
         class_converter = fhir_utils.fhir_specimen
 
     def create(self, validated_data):
-        procedure_data = validated_data.pop('procedure')
-        procedure_model, _ = Procedure.objects.get_or_create(**procedure_data)
-        biosample = Biosample.objects.create(procedure=procedure_model, **validated_data)
+        biosample = Biosample.objects.create(**validated_data)
         return biosample
 
     def update(self, instance, validated_data):
         instance.sampled_tissue = validated_data.get('sampled_tissue', instance.sampled_tissue)
-        instance.individual_age_at_collection = validated_data.get('individual_age_at_collection',
-                                                                   instance.individual_age_at_collection)
+        instance.time_of_collection = validated_data.get('time_of_collection', instance.time_of_collection)
```

Comment:
maybe update() can be tested, although if it wasn't before then maybe it isnt needed for now

Reply:
tested with an api PUT method